### PR TITLE
[WIP] Translation: desktop-client/src/components/budget/BudgetTotals.tsx

### PR DESCRIPTION
**Pull request by nmathey**
Translation support for: desktop-client/src/components/budget/BudgetTotals.tsx

### CODE REVIEW

File: packages/desktop-client/src/components/budget/BudgetTotals.tsx [hunk 1]
```diff
@@ -54,7 +56,7 @@ export const BudgetTotals = memo(function BudgetTotals({
           WebkitUserSelect: 'none',
         }}
       >
-        <View style={{ flexGrow: '1' }}>Category</View>
+        <View style={{ flexGrow: '1' }}>{t('Category')}</View>
```
Comment (jfdoming) [inline packages/desktop-client/src/components/budget/BudgetTotals.tsx:59]: We should also translate the `aria-label` and the `text` strings below

Comment (nmathey) [reply packages/desktop-client/src/components/budget/BudgetTotals.tsx:59]: `aria-label={t('Menu')}` should work?

Comment (jfdoming) [reply packages/desktop-client/src/components/budget/BudgetTotals.tsx:59]: Yep!

Comment (jfdoming) [reply packages/desktop-client/src/components/budget/BudgetTotals.tsx:59]: And same for `text`

Comment (nmathey) [reply packages/desktop-client/src/components/budget/BudgetTotals.tsx:59]: mmm got `Type '{ t(: any): any; }' is not assignable to type 'string'.` while doing it for `text` :/

Comment (nmathey) [reply packages/desktop-client/src/components/budget/BudgetTotals.tsx:59]: in this case don't need `{ }` I guess.

File: packages/desktop-client/src/components/budget/BudgetTotals.tsx [hunk 2]
```diff
@@ -54,11 +56,11 @@ export const BudgetTotals = memo(function BudgetTotals({
...
```
Comment (jfdoming) [inline packages/desktop-client/src/components/budget/BudgetTotals.tsx:59]: Looks good now! Sorry for one last comment, the convention we've been using in the rest of the code base for components is to use `<Trans>`. I.e.:
```suggestion
        <View style={{ flexGrow: '1' }}><Trans>Category</Trans></View>
```
You'll need to import it as well

Comment (nmathey) [reply packages/desktop-client/src/components/budget/BudgetTotals.tsx:59]: Yeah there are discussion around those to different stage on discord. I did see real statement yet but I'm might be wrong, that why I use the lazy way using one unique code style :p 

However reading the doc it is clearer for me I have to use <Trans> component in some cases: https://actualbudget.org/docs/contributing/i18n

I'll rework the file. Thanks for your heads up.

**Review by jfdoming — APPROVED** [review]

**Pull request merged**